.github/workflows/release.yml

```diff
@@ -7,15 +7,15 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v4
       with:
-        go-version: 1.16
+        go-version: "1.20"
```

Comment:
Should we use 1.18 instead?

Reply:
Thank you @xieshenzh .
I think we can keep it 1.20, this sets the version we build with, this shouldn't affect the target version.

We currently targeting 1.15, if we want, we can bump it to 1.18 [here](https://github.com/fabric8-analytics/cli-tools/blob/main/go.mod#L3), but probably it'll be better to do it in a different PR.

Reply:
We can discuss the target version offline, so we can be sure that's what we want to do.
Regardless, the version we build with, shouldn't have any affect.

Reply:
@xieshenzh 
When I think about it, you made a good point. Especially post ADR-0006...

While it's ok to build and test everything with 1.20.
The `gomanifest`  tool should be verified with 1.15, but I don't see we have any test or workflow handling this.

I'm gonna put this PR on hold until I'll dd a workflows running 1.15 and verifying `gomanifest`.

Reply:
@xieshenzh 
I added a workflow for building the `gomanifest` tool on multiple operating systems using go 1.15.
![image](https://user-images.githubusercontent.com/28388442/230845858-db52560a-57de-43f1-bdcd-b8b3f8317a99.png)

> The Build GoManifest job is running 1.15.